Formula/tt.rb

```diff
@@ -29,6 +29,7 @@ def install
     system "mage", "build"
     bin.install "tt"
     (etc/"tarantool").install "tt.yaml.default" => "tt.yaml"
+    generate_completions_from_executable(bin/"tt", "completion", base_name: "tt", shells: [:bash, :zsh])
```

Comment:
Since the formula's name already is `tt`, the `base_name` can be omitted

Reply:
Fixed, thank you!